pkg/minishift/config/hostfolder.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"fmt"
+)
+
+type HostFolder struct {
+	Name    string
+	Type    string
+	Options map[string]string
+}
+
+func (hf *HostFolder) Mountpoint() string {
+	return fmt.Sprintf("/mnt/sda1/%s", hf.Name)
```

Comment:
I might be missing something, but how do I get a my host folder C:\Users\Hardy mounted under /c/Users/Hardy?